services/galley/galley.integration.yaml

```diff
@@ -23,7 +23,7 @@ spar:
 settings:
   httpPoolSize: 128
   maxTeamSize: 32
-  truncationLimit: 24
+  truncationLimit: 18
```

Comment:
I guess cailleach PR needs to be updated now: https://github.com/zinfra/cailleach/pull/334/